### DISCUSSION TITLE
PHP_Realtime_Chat_App Discussions!

### DISCUSSION THREAD

**Discussion by tanhongit** [tanhongit/PHP_Realtime_Chat_App#5] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by poiuto** [comment, marked as answer]
yo?

**Reply by tanhongit** [reply]
Good day poiuto

**Comment by tanhongit** [comment]
Good day **poiuto**, 
Do you have a problem?
